scenariolib/event_pageview.go

```diff
@@ -9,16 +9,17 @@ import (
 // ============== VIEW EVENT ======================
 // =================================================
 
-// ViewEvent a struct representing a view, it is defined by a clickRank, an
-// offset, a probability to click, the contentType and a pageViewField
-// We keep a similar structure to a click event because we can simulate that the user
-// visited a page that was returned as a result
+// ViewEvent a struct representing a view, it is defined by a clickRank, an offset, a probability to click,
```

Comment:
There are no clickRank in page views, so I am not sure that this manke sense here

Reply:
Yes, I know, but I treat the view event "as a click" for the random data generation purposes.
Basically for my bot it doesn't make sense to send a view event on something that is not in your index, so you search first then "click/view" one of the result as to have recommendations ultimately.

I know it's confusing, took me a while to remember why I had done it that way too. I think I will document/comment the reasons why.

(The clickRank allows you to do a specific search then "view" a specifically ranked result).